src/ui/widgets/EmbeddedDisplay/bobParser.ts

```diff
@@ -210,8 +210,8 @@ function bobParseResizing(jsonProp: ElementCompact): string {
 
 function bobParseSymbols(jsonProp: ElementCompact): string[] {
   const symbols: string[] = [];
-  jsonProp["symbol"].forEach((item: any) => {
-    symbols.push(item._text);
+  Object.values(jsonProp["symbol"]).forEach((item: any) => {
+    symbols.push(item);
```

Comment:
I am getting a parsing error with this change, which I wasn't getting before.
```
Error: Failed widget description type: Invalid widget description `symbols[0]` of type `object` supplied to `Symbol$1`, expected `string`.
```
It worked when I put it back to `item._text`?

Reply:
Thanks for spotting this! I've added an extra check to see if we receive an object or a string and parse it accordingly, to account for single symbol and multiple symbol widgets.